code/src/components/Tvseries.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { TV_SERIES_URL } from 'utils/urls';
+
+export const Tvseries = () => {
+  const [tvList, setTvList] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    fetch(TV_SERIES_URL)
+      .then((response) => response.json())
+      .then((data) => setTvList(data.results))
+      .catch((error) => alert(error, 'error'))
+      .finally(() => setLoading(false));
+  }, [])
+
+  if (loading) {
+    return (
+      <div className="loading-container">
+        <h2>Loading..</h2>
+      </div>
+    );
+  }
+
+  return (
+    <section className="tvListWrapper">
+      <h1 className="popular-tv">Popular TV-Series</h1>
+      <div className="popular-tv-container">
+        {tvList.map((singleTv) => {
+          return (
+            <div className="tv-wrapper">
```

Comment:
Might I suggest making it easier for you if you use the same classnames on these divs as in the movie parts? That way you don't have to apply css duplicates